### PR TITLE
use git to build sipnet

### PR DESCRIPTION
**Pull request by robkooper**
This changes the dockerfile and some scripts to use the version that is in git of sipnet. This will install both the latest and r136 version of sipnet (currently these are the same).

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: models/sipnet/Dockerfile [hunk 1]
```diff
@@ -48,13 +47,10 @@ FROM pecan/models:${IMAGE_VERSION}
...
-ARG MODEL_VERSION=136
-
 # Setup model_info file
 COPY model_info.json /work/model.json
 RUN sed -i -e "s/@VERSION@/${MODEL_VERSION}/g" \
            -e "s#@BINARY@#/usr/local/bin/sipnet.${MODEL_VERSION}#g" /work/model.json
 
 # COPY sipnet binary
-COPY --from=sipnet-binary /src/sipnet /usr/local/bin/sipnet.${MODEL_VERSION}
+COPY --from=sipnet-binary /src/sipnet/sipnet /usr/local/bin/sipnet.${MODEL_VERSION}
```
Comment (ashiklom) [inline models/sipnet/Dockerfile]: Is this a typo, or does the model install itself into a `sipnet` subdirectory of its own repo?

Comment (robkooper) [reply models/sipnet/Dockerfile]: nope, the model when compiled creates a binary that is called sipnet in the sipnet folder. When finished compiling I will copy the binary from one container to another container using this copy command.

Comment (ashiklom) [reply models/sipnet/Dockerfile]: Ah, OK, thanks!

File: scripts/install_pecan.sh [hunk 2]
```diff
@@ -396,29 +396,24 @@ sudo cp ed_2.1-opt /usr/local/bin/ed2.git
...
-  curl -o sipnet_unk.tar.gz http://isda.ncsa.illinois.edu/~kooper/PEcAn/sipnet/sipnet_unk.tar.gz
-  tar zxf sipnet_unk.tar.gz
-  rm sipnet_unk.tar.gz
+  git clone https://github.com/PecanProject/sipnet.git
+  sed -i 's#$(LD) $(LIBLINKS) \(.*\)#$(LD) \1 $(LIBLINKS)#' ${HOME}/sipnet/Makefile
 fi
-cd ${HOME}/sipnet_unk/
+
+cd ${HOME}/sipnet/
+git checkout r136
```
Comment (istfer) [inline scripts/install_pecan.sh]: do we need a `r136` branch in sipnet repo for this?

Comment (robkooper) [reply scripts/install_pecan.sh]: it is a tag.

Comment (istfer) [reply scripts/install_pecan.sh]: got it!

**Pull request merged**